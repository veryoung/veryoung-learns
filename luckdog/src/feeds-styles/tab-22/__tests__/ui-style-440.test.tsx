import React from 'react';
import renderer from 'react-test-renderer';

import FeedsViewUIStyle440 from '../ui-style-440';


describe('<FeedsViewUIStyle440/>', () => {
  describe('render testing', () => {
    it('render style440', () => {
      const props = {
        itemBean: {
          parsedObject: {
            sData: '',
          },
        },
        index: 0,
      };
      const tree = renderer.create(<FeedsViewUIStyle440 {...props} />);
      expect(tree).toMatchSnapshot();
    });
  });
});
