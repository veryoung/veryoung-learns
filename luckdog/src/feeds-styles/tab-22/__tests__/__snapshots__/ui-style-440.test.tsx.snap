// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeedsViewUIStyle440/> render testing render style440 1`] = `null`;
